### ISSUE TITLE
Geänderter Wert bei "RGBW_COLOR|WHITE_HUE_VALUE" wird nicht dauerhaft übernommen

### ISSUE THREAD

**Issue by AdlerCentauri** [thkl/Homematic-Virtual-Interface#58]
Bei den von dem Hue Plugin angelegten HM-LC-RGBW-WM Geräten, werden die Änderungen bei "RGBW_COLOR|WHITE_HUE_VALUE" nicht dauerhaft übernommen. Wenn man den Wert ändert, dann ist er auch aktiv gespeichert und die Leuchten haben dann den geänderten Weiß Wert, aber dieser geht nach einer Zeit verloren und es ist wieder der Standardwert 39609 gesetzt.

**Comment by AdlerCentauri** [comment]
Stimmt, tatsächlich ist es so, dass sich der Wert erst nach einigen Stunden ändert.

**Comment by thkl** [comment]
Ich glaub das ja , aber ich kann es nicht ganz nachvollziehen. Das Plugin fragt die Hue Lampen aller X Sekunden ab und berechnet dann die Homematic Werte aus den aktuellen Werten der Lampe.
Warum das nach x Stunden kaputt gehen soll verstehe ich nicht ganz.
Kann mal jemand probieren, ob das auch anderweitig getriggert wird (Lampe aus oder ähnlich)

**Comment by AdlerCentauri** [comment]
Hi @thkl , also ich meine nicht den Dimmwert oder den Wert der Farbe. Sondern den in den Einstellungen.

Einstellungen > Geräte (HVL)-HM-LC-RGBW-WM > Einstellen > RGBW_COLOR|WHITE_HUE_VALUE
<img width="859" alt="bildschirmfoto 2018-11-27 um 20 18 28" src="https://user-images.githubusercontent.com/37038767/49105741-bec3b500-f281-11e8-8bb8-32e2eb968d42.png">
Wenn ich den Wert "39609" auf "9609" ändere, dann ist er nach ein Paar stunden wieder von alleine auf "39609" gesetzt.

**Comment by thkl** [comment]
achso , ok ich schau mal ob ich da irgendwo zu oft defaults pushe

**Comment by AdlerCentauri** [comment]
Hallo @thkl könntest Du dir das bitte zeitnah ansehen. Danke dir!

**Comment by AdlerCentauri** [comment]
@thkl warum wird das jetzt ignoriert?

**Comment by thkl** [comment]
Könnte daran liegen, das wenig Zeit dafür ist

**Comment by AdlerCentauri** [comment]
Ok, das Problem ist zwar nervig. Aber dann muss man erstmal damit leben. Danke trotzdem.
